tests/test_routes.py

```diff
@@ -44,6 +48,64 @@ def test_create_application_is_successful(client):
     count_fund_applications(client, "fund-b", expected_length_fund_b)
 
 
+def test_create_application_creates_formatted_reference(client):
+    """
+    GIVEN We have a functioning Application Store API
+    WHEN we try to create an application
+    THEN a correctly formatted reference is created for the application
+    """
+    create_application_json = {
+        "account_id": "usera",
+        "fund_id": "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4",
+        "round_id": "c603d114-5364-4474-a0c4-c41cbf4d3bbd",
+    }
+    response = client.post(
+        "/applications",
+        data=json.dumps(create_application_json),
+        content_type="application/json",
+        follow_redirects=True,
+    )
+    application = response.json
+    assert application["reference"][:8] == "COF-SUM-"
```

Comment:
⛏️ You can use `startswith` and `endswith` for a slightly (very slightly) cleaner syntax.
https://docs.python.org/3/library/stdtypes.html#str.endswith